_includes/navigation.html

```diff
@@ -0,0 +1,5 @@
+ {% for item in site.data.navigation %}
+    <a href="{{ item.link }}" class="menulink" {% if page.url == item.link %}{% endif %}>
+      {{ item.name }}
+    </a>
+  {% endfor %}
```

Comment:
Prüfe hier bitte, dass am Schluss ein Zeilenumbruch steht. Am besten konfigurierst du deine IDE so, dass sie den Zeilenumbruch automatisch einfügt, wenn du eine Datei speicherst.

Die Einrückung sieht auch nicht ganz richtig aus: `{% endfor %}` sollte auf der gleichen Ebene sein wie `{% for item in site.data.navigation %}`.

Reply:
da wollte ich dich eh was fragen, ich kann da ja genauso mit Prettier arbeiten oder? ich habe an und für sich eingestellt, dass beim Speichern immer mit Prettier das Dokument formatiert werden soll und mir ist aufgefallen, dass es das nicht tut...ich werde mal schauen ob das überhaupt geht

Reply:
Ja, das wäre super! Das mit dem Zeilenumbruch geht aber auch ohne Prettier. Such im VSCode mal nach den folgenden Settings und aktiviere sie:

<img width="645" alt="Bildschirmfoto 2023-10-30 um 09 39 46" src="https://github.com/DevCraft-Academy/landingpage/assets/141632/83ba3f3c-6418-43f2-b4ba-085358d33ff0">

Das stellt einerseits sicher, dass eine Newline am Ende eingefügt wird, andererseits vermeidet es auch mehrere Newlines am Ende der Datei.